### PR TITLE
Eng 15692

### PR DESCRIPTION
**Pull request by manjujames**

### CODE REVIEW

File: src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java [hunk 1]
```diff
@@ -199,7 +190,7 @@ private void checkSiteConsistency(VoltTableRow row) throws IOException
 
     private SynthesizedPlanFragment
     constructDistributeReplicatedTableAsReplicatedFragment(long sourceSiteId,
-                                                           long destinationSiteId)
+                                                           int destinationSiteId)
```
Comment (yzhang1991) [inline src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java]: destHostId

File: src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java [hunk 2]
```diff
@@ -222,6 +213,7 @@ private void checkSiteConsistency(VoltTableRow row) throws IOException
     constructLoadReplicatedTableAggregatorFragment(boolean asPartitioned)
     {
         int result_dependency_id = getNextDependencyId();
+        //setRootDependencyId(resultDependencyId);
```
Comment (yzhang1991) [inline src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java]: Can delete this?

File: src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java (hunk 1 above)
Comment (manjujames) [reply src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java]: OK

File: src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java (hunk 2 above)
Comment (manjujames) [reply src/frontend/org/voltdb/sysprocs/saverestore/ReplicatedTableSaveFileState.java]: OK, will do. This was in 8.4 codebase and I wasn't sure if this was required.

**Pull request merged**